supplier_app/models.py

```diff
@@ -337,8 +337,17 @@ class Address(models.Model):
         verbose_name=_("Country"),
     )
     taxpayer = models.ForeignKey(TaxPayer, on_delete=models.CASCADE)
+    history = HistoricalRecords()
+
 
+class ContactInformation(models.Model):
+    address = models.ForeignKey(Address, on_delete=models.CASCADE)
```

Comment:
Is ContactInformation going to use an address different from the one taxpayer has.